### PR TITLE
Add release notes for v1.5.0

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Signed-off-by: Denis Tingaikin <denis.tingajkin@xored.com>

### CODE REVIEW

File: static/modules/repo-map.js [hunk 1]
```diff
@@ -21,13 +21,15 @@
...
+        "cmd-nse-l7-proxy": "cell-vHwpXIBINaUL7lCKyK2X-1"
       },
       "sdk-vpp": {
         "cmd-nsc-vpp": "cell-KUktbazg4igNnRH6aAHL-45",
         "cmd-nse-icmp-responder-vpp": "cell-KUktbazg4igNnRH6aAHL-44",
         "cmd-nse-firewall-vpp":"cell-YWTRKo84t5DHiULYztLq-3",
         "cmd-forwarder-vpp": "cell-KUktbazg4igNnRH6aAHL-15",
-        "cmd-nse-simple-vl3-docker":"cell-tK4_8g8Ls3PgPuG9fXJk-1",
+        "cmd-nse-simple-vl3-docker":"cell-pbsQXEyPZ_YmtPOauGdX-13",
+        "nsc-simple-docker":"cell-pbsQXEyPZ_YmtPOauGdX-14",
```
Comment (glazychev-art) [inline static/modules/repo-map.js:32]: Do we need to use `cmd-nsc-simple-docker` instead of `nsc-simple-docker`?

**Comment by glazychev-art** [comment]
A few comments:
`cmd-nse-simple-vl3-docker` is listed twice
`cmd-nse-remote-vlan` is listed twice
wrong link for `cmd-forwarder-sriov`
`cmd-cluster-info-k8s` depends on sdk

**Comment by denis-tingaikin** [comment]
@glazychev-art Thanks!


Should be resolved.

File: content/docs/releases/v1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+### Added support for VMs/Bare Metal external clients
+
+Developed an application to allow for non k8s clients connect to NSM infrastructure from outside.
+
+[More Details](https://github.com/networkservicemesh/cmd-nsc-simple-docker/issues/1) 
+[Try it!](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.5.0/examples/k8s_monolith/external_nsc)
+
+
+### Distributed DNS over vl3 network
```
Comment (fkautz) [inline content/docs/releases/v1.5.0.md:44]: We should capitalize the L for vL3

**Review by fkautz — CHANGES_REQUESTED** [review]

File: content/docs/releases/v1.5.0.md (hunk 2 above)
Comment (denis-tingaikin) [reply content/docs/releases/v1.5.0.md]: Sure! Fixed.

**Review by fkautz — APPROVED** [review]

**Pull request merged**